src/WebView.cpp

```diff
@@ -7,9 +7,6 @@
 namespace
 {
     constexpr auto const WHATSAPP_WEB_URI        = "https://web.whatsapp.com/";
-    constexpr auto const SCRIPT_DARK_MODE        = "document.body.className=\"web dark\"";
-    constexpr auto const SCRIPT_REVERT_DARK_MODE = "document.body.className=\"web\"";
-
 
     void loadChanged(WebKitWebView* webView, WebKitLoadEvent loadEvent, gpointer userData)
```

Comment:
Could you please remove this function and related signal binding since we don't need it anymore.

Reply:
Done.